backend/src/v5/utils/helper/images.js

```diff
@@ -15,21 +15,54 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+const { createCanvas } = require('@napi-rs/canvas');
 const sharp = require('sharp');
 
 const ImageHelper = {};
 
-ImageHelper.createThumbnail = async (buffer, width = 600, density = 150) => {
+let pdfJsLibCache;
+
+const loadPdfJsDist = () => new Promise((resolve) => {
+	if (!pdfJsLibCache) {
+		// eslint-disable-next-line node/no-unsupported-features/es-syntax, import/extensions
+		import('pdfjs-dist/legacy/build/pdf.mjs').then((pdfJsLib) => {
+			pdfJsLibCache = pdfJsLib;
+			resolve(pdfJsLibCache);
+		});
+	} else {
+		resolve(pdfJsLibCache);
+	}
+});
+
+ImageHelper.createThumbnail = async (buffer, mimeType, width = 600, density = 150) => {
 	if (!buffer) throw new Error('Image not provided');
 
-	const jpgBuffer = await sharp(buffer, { density })
-		.flatten({ background: '#ffffff' })
-		.toFormat('jpeg')
-		.toBuffer();
+	let imgBuffer;
+	if (mimeType === 'application/pdf') {
```

Comment:
can we put the mimeTypes to a constant file? maybe in `/utils/helpes/typeCheck.constants` (thought we had one already but it looks like we don't! 🙈